src/poliastro/core/propagation/farnocchia.py

```diff
@@ -283,6 +283,17 @@ def nu_from_delta_t(delta_t, ecc, k=1.0, q=1.0, delta=1e-2):
     return nu
 
 
+@jit
+def farnocchia_coe(k, p, ecc, inc, raan, argp, nu):
+    
+    q = p / (1 + ecc)
+
+    delta_t0 = delta_t_from_nu(nu, ecc, k, q)
+    delta_t = delta_t0 + tof
```

Comment:
Woops, I forgot to specify in the issue that we're missing a `tof` argument for `farnocchia_coe`, at the end

Reply:
ohh! My bad! I fixed it and pushed the changes